.git-blame-ignore-revs

```diff
@@ -0,0 +1,50 @@
+# Pinking shears
+a4789b3cc1cb59e5cd062670f4439f264c0d34f3 # [12042]
+fb4e38b0e750c0cab5eb4c50532da93e2d9882b3 # [15843]
+4b33a0e9c45d67c5f3123577262f73727444254a # [16438]
+fa33255b567df6b19ccc3a61824ad243db116993 # [16439]
+46d96c7704b978b9d070bfec1cb1a33e4caf46b0 # [18254]
+feaf2597bf16b1bd6b489a0e5680501f12990bee # [18276]
+9e7890c3f61781200b9337186d85cd4034edac61 # [18386]
+fd10e500e47ca1ae3280ac6817e725965f7fc0dc # [19054]
+6610e321e79d9b56c6fc496b6630dd2ac664a0d3 # [19528]
+d26f0a4c407a5d13a095c7676b397e51a959d82c # [19577]
+89e9bcc1d69ebbac01c8c4501f9066e421125e66 # [20000]
+8c50f982ea9784fe06844ce84b1a3297b1241442 # [20715]
+b6e23d7269e50296270f93dda96dacb118bee8b3 # [20944]
+48a3ec24c5ea7615431bde431098546674a0096c # [21070]
+2439e4d722f5093631c4bac9a45106ab33624aaa # [21381]
+56c4577feae624f7a3bf55d001295e0707c972b6 # [21486]
+680b671330327f2568549dec9460a2e942219d55 # [21492]
+dd39a3b3d2d1f792b6ed8e88b1fcc13877fbd8dd # [22491]
+6d8bce688f5215ad20133b2cefedc72ac5136be3 # [22634]
+05c0f14024a2bb8ef9bdfcb84589d04205f8ce24 # [23679]
+ece7a7714477142d4cdb3269d2f9ad3d5dab2e5f # [23780]
+687d1a2ce992a0e7e24675b506b85a57e42fe78e # [24303]
+45d2a20783460667f85d5fdced422efd54c33b72 # [24603]
+c8889d984fd98381d6f0bd3228972e93348f1266 # [25085]
+21da24227f2c0087c95f44c60b1bca65cedf0611 # [25824]
+30f822b8ee767c0b07fe1780f2829de0cba8c314 # [25880]
+1cdb0ac2fa5452e58d7a002b260d594a581af02e # [26475]
+37a37b5fc578d945376cf3b66b4a23265d6491f3 # [26597]
+bda43fd1071ed45ded3def7872866f62e40af9f5 # [26627]
+98fe4a5aedfdaa8c769bf2aaedc58eb63d342320 # [26631]
+d6e06a2ee5c15b19ca80b9162f3e19c06b5af34d # [26714]
+cfd5c395bb3ba810edaec6cf4483212181823d2e # [26851]
+2ec5e68249bdacee9ac0ce59fad91b1149244af8 # [27123]
+305e72859a4154fc96bac7774ab7808ead6a06d4 # [29169]
+ca32a2d410f954b6b40d23f3392ba53e62a3d9f1 # [29707]
+48a504cc50a16f60a395ace8d133130e3d6962f5 # [30047]
+b539bd985bdf2dde162f46fcbf14e51e46ff8be7 # [30372]
+9a3942ffd82b28a77982e0a0d1c2bb0b8a70ce44 # [30996]
+469164785ff8defefbfcdf972c57d9004c273256 # [31077]
+c4e9c64233166957cd1cbcc54ff54b1a02edd3b5 # [31623]
+f4f1b4821342fd1d58708356b3ec39d6fefe31ac # [33411]
+0ec540b946eaebebc7e8ee39a8e4738914e7f2a3 # [33627]
+0c5bd752629b0960c6f43ca212a64724e4f40346 # [34534]
+991feb70438e981290696fa2b4fbe2ea54696a02 # [34774]
+6911ff11308089eace23719fc50160d403081a8e # [35627]
+8df8cf2df14fe26174f97af5bb17d63f2e867231 # [42843]
+
+# PHPCS
```

Comment:
```suggestion
# Coding Standards
```

A nit. I think it fine to group PHP, JS and CSS coding standards fixes in the one spot.

Reply:
Sure, maybe if/as it grows there can be subheadings or split back out.